### PR TITLE
Admin/manage areas/edit registered areas

### PR DESCRIPTION
**Pull request by YoshimiHELLo**
[admin/manageAreas/editRegisteredAreas] - edit fees UI
Additions:
* Added the area fees UI in edit.blade.php(admin/areas).
* Added a route in web.php
* Added function in AreasController.php

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -33,4 +35,5 @@
     Route::get('/attribute/edit', [AttributesController::class, 'editAttribute'])->name('admin.attributes.edit');
     Route::get('/showusers', [UsersController::class, 'showUsers'])->name('showusers');
     Route::get('/admins/register', [AdminsController::class, 'registerAdmin'])->name('admins.register');
+    Route::get('/areas/edit',[AreasController::class,'editRegisteredAreas'])->name('admin.areas.edit');
```
Comment (KobeSergio) [inline routes/web.php]: Please see comments on PR#17 first regarding the redundant admin route :) 

`->name('admin.areas.edit');` will look something like `admin.admin.areas.edit` ❌

**Review by KobeSergio — CHANGES_REQUESTED** [review]

File: routes/web.php (hunk 1 above)
Comment (YoshimiHELLo) [reply routes/web.php]: @KobeSergio 
Thank you. I've changed it to ->name('areas.edit');.
Is it OK with it?

Comment (KobeSergio) [reply routes/web.php]: Yes, its okay now :)

**Comment by KobeSergio** [comment]
@YoshimiHELLo please resolve conflict again, there has been a conflict with PR 18

**Comment by YoshimiHELLo** [comment]
@KobeSergio 
I resolve a conflict, kindly review it please.

**Pull request merged**